rueidisaside/aside.go

```diff
@@ -144,14 +144,22 @@ func randStr() string {
 func (c *Client) Get(ctx context.Context, ttl time.Duration, key string, fn func(ctx context.Context, key string) (val string, err error)) (string, error) {
 	ctx, cancel := context.WithTimeout(ctx, ttl)
 	defer cancel()
+
+	useLuaScript := false // @rueian how do we configure this?
```

Comment:
@rueian How do you suggest we do this?

Reply:
Add the option after https://github.com/redis/rueidis/blob/34c5717db2b362f2fb94ea5fef54283bf14f9113/rueidisaside/aside.go#L21.

Maybe name it `UseLuaLock bool`.